test/unit/account/states_test.rb

```diff
@@ -206,21 +206,21 @@ def test_suspend_master
 
   test '.inactive_since' do
     inactive_since_days = 5.days
+    provider = FactoryBot.create(:provider_account)
 
-    old_account_without_traffic = FactoryBot.create(:simple_account)
+    old_account_without_traffic = FactoryBot.create(:buyer_account, provider_account: provider)
     old_account_without_traffic.update_attribute(:created_at, inactive_since_days.ago)
 
-    old_account_with_old_traffic = FactoryBot.create(:simple_account)
+    old_account_with_old_traffic = FactoryBot.create(:buyer_account, provider_account: provider)
     old_account_with_old_traffic.update_attribute(:created_at, inactive_since_days.ago)
-    FactoryBot.create(:cinstance, user_account: old_account_with_old_traffic, first_daily_traffic_at: inactive_since_days.ago)
+    FactoryBot.create(:cinstance, user_account: old_account_with_old_traffic, first_daily_traffic_at: inactive_since_days.ago - 1)
```

Comment:
I don't know how this worked in the past. Maybe a second passed because of slower factories previously and it eventually worked. Or maybe test executor is slower so that a second passes.

But according to code, there should be no traffic since the inactive time. To be sure that works, we need to be at least a second before it.